.circleci/config.base.yml

```diff
@@ -1,1560 +1,1560 @@
-version: 2.1
-orbs:
-  aws-ecr: circleci/aws-ecr@6.15.3
-machine:
-  environment:
-    PATH: '${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin'
+# version: 2.1
+# orbs:
```

Comment:
disabling CCI should be separate PR.

Reply:
Yep, we shouldn't turn off CCI until we've done at least a couple releases using CodeBuild